getting-started/setup-building.rst

```diff
@@ -655,3 +655,72 @@ every rule.
 
 
 .. _issue tracker: https://github.com/python/cpython/issues
+
+
+.. _using-codespaces:
+
+Contribute using GitHub Codespaces
+==================================
+
+.. _codespaces-whats-codespaces:
+
+What is GitHub Codespaces?
+--------------------------
+set up the same environment but locally in a docker container if you'd prefer th
```

Comment:
There's something off with the start of this sentence :)

Reply:
It can be deleted, the full sentence is at line 682 below :)